src/components/ProtVista/index.js

```diff
@@ -929,16 +931,24 @@ export class ProtVista extends Component /*:: <Props, State> */ {
                       className={f('conservation-placeholder-component')}
                       ref={this._conservationTrackRef}
                     >
-                      {this.state.showLoading ? (
-                        <div
-                          className={f('loading-spinner')}
-                          style={{ margin: '10px auto' }}
-                        >
-                          <div />
-                          <div />
-                          <div />
+                      {this.props.conservationError ? (
+                        <div className={f('conservation-error')}>
+                          ⚠️ {this.props.conservationError}
                         </div>
-                      ) : null}
+                      ) : (
+                        <>
+                          {this.state.showLoading ? (
+                            <div
+                              className={f('loading-spinner')}
+                              style={{ margin: '10px auto' }}
+                            >
+                              <div />
+                              <div />
+                              <div />
+                            </div>
```

Comment:
Is it not possible to import and use `Loading` from `components/SimpleCommonComponents/Loading`?